gitops/git/git.go

```diff
@@ -103,6 +104,34 @@ func (r *Repo) Commit(message, gitopsPath string) bool {
 	return true
 }
 
+// RemoveDiff removes the changes made to a specific file in the repository
+func (r *Repo) RemoveDiff(fileName string) {
+	exec.Mustex(r.Dir, "git", "checkout", "--", fileName)
+}
+
+// split by newline and ignore empty strings
+func SplitFunc(c rune) bool {
```

Comment:
This function doesn't seem to belong to the public interface of `git` module. Please make it private or inline.